rpc/manager/manager.proto

```diff
@@ -262,6 +262,26 @@ message LogLevelRequest {
   google.protobuf.Duration duration = 2;
 }
 
+message GetLogsRequest {
+  // Whether or not logs from the traffic-manager are desired
+  bool traffic_manager = 1;
+
+  // The traffic-agent(s) logs are desired from. Can be `all`, `False`,
+  // or substring to filter based on pod names.
+  string agents = 2;
+}
+
+message LogsResponse {
+
+  // The first value of this map is the `podName.namespace` and the
+  // second value is the logs from that pod
```

Comment:
It's confusing to talk about first and second value in a map because the map is unordered. I think you mean:
The map contains associations between \<podName.namespace> and logs from that pod.

Reply:
Yeah I was clearly struggling with how to word this and what you have is much better